views/form.html

```diff
@@ -27,77 +27,61 @@ <h2 class="prospect-form__heading">And get 30 more days of free FT</h2>
 				<p>{{error}}</p>
 			</div>
 			{{/if}}
-		
-			<div class="o-forms">
-				<label for="prospect-form__first-name" class="o-forms__label">First name</label>
-				<input name="firstName" data-trackable="first-name" type="text" id="prospect-form__first-name" placeholder="Enter your first name" class="o-forms__text" required>
-				<div class="o-forms__errortext">Please enter your first name.</div>
-			</div>
-			<div class="o-forms">
-				<label for="prospect-form__last-name" class="o-forms__label">Last name</label>
-				<input name="lastName" data-trackable="last-name" type="text" id="prospect-form__last-name" placeholder="Enter your last name" class="o-forms__text" required>
-				<div class="o-forms__errortext">Please enter your last name.</div>
-			</div>
-			<div class="o-forms">
-				<label for="prospect-form__job-title" class="o-forms__label">Job title</label>
-				<input name="title" data-trackable="job-title" type="text" id="prospect-form__job-title" placeholder="Enter your job title" class="o-forms__text" required>
-				<div class="o-forms__errortext">Please enter your job title.</div>
-			</div>
-			<div class="o-forms">
-				<label for="prospect-form__company-name" class="o-forms__label">Company name</label>
-				<input name="company" data-trackable="company-name" type="text" id="prospect-form__company-name" placeholder="Enter your company name" class="o-forms__text" required>
-				<div class="o-forms__errortext">Please enter your company name.</div>
-			</div>
+
+			{{> n-conversion-forms/partials/firstname }}
+
+			{{> n-conversion-forms/partials/lastname }}
+
+			{{> n-conversion-forms/partials/job-title }}
+
+			{{> n-conversion-forms/partials/company-name }}
+
 			{{#unless isUnmasking}}
-			<div class="o-forms">
-				<label for="prospect-form__work-email" class="o-forms__label">Work email address</label>
-				<input name="email" data-trackable="email-address" type="email" id="prospect-form__work-email" placeholder="Enter your work email address" class="o-forms__text" required>
-				<div class="o-forms__errortext">Please enter your work email address.</div>
-			</div>
+			{{> n-conversion-forms/partials/email label="Work email address" }}
 			{{/unless}}
-			<div class="o-forms">
-				<label for="prospect-form__phone-number" class="o-forms__label">{{#if isUnmasking}}Phone{{else}}Work phone{{/if}} number</label>
-				<small class="o-forms__additional-info">5 to 15 characters (numbers only)</small>
-				<input name="phone" data-trackable="phone-number" type="text" id="prospect-form__phone-number" placeholder="Enter your phone number" class="o-forms__text" required pattern="([0-9()+\- ])+" minlength="5" maxlength="15">
-				<div class="o-forms__errortext">Please enter a valid work phone number.</div>
-			</div>
+			
+			{{> n-conversion-forms/partials/phone label="Work phone number" }}
```

Comment:
Should we pass through labels like this or use a `isB2b` check -

Pros 
 * Very explicit which forms use which copy
 * Keeps copy in one place

Cons
 * Less customisable
 * Could end up with lots of `is*` params

Reply:
I'd pass through the labels, I can see that it'd be cleaner to have to copy in one place but I've also seen cases of if X -> if Y -> if Z -> Use copy A situations...

Reply:
I've just moved it to using flags :(

Over the weekend I was thinking what would be most useful and came to the conclusion that to keep consistency on the forms having the copy in one place makes sense.

It also means variations can be kept in check as we will know all of them. If someone wants to change the copy it will either change across every form or they need to provide a good reason for us introducing the variation.